hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -1,29 +1,29 @@
 # NVMe SSD Boot (BETA)
 
-NVMe (Non Volatile Memory express) is a standard for accessing Solid State Drives (SSD) via a PCI express bus (PCIe). You can connect these drives via the PCIe slot on a Compute Module (CM) IO board allowing a CM4 to boot from SSD.
+NVMe (non-volatile memory express) is a standard for accessing solid state drives (SSDs) via a PCIe bus. You can connect these drives via the PCIe slot on a Compute Module (CM) IO board, allowing a CM4 to boot from SSD.
```

Comment:
Aren't the capitals appropriate here, as they help explain the acronyms?

Reply:
The style guide says to not use initial caps when expanding initialisms and acronyms.

Reply:
Fair enough.